easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.36-GCC-9.3.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'DIAMOND'
+version = "0.9.36"
+
+homepage = 'https://github.com/bbuchfink/diamond'
+description = "Accelerated BLAST compatible local sequence aligner"
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/bbuchfink/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['870af89606330f6c0b57ed478d2d8237334ce8f5630eac770399ff431948bd59']
+
+separate_build_dir = True
+
+builddependencies = [
+    ('binutils', '2.34'),
```

Comment:
@mboisson Existing DIAMOND easyconfigs also list `zlib` as a dependency, which was dropped here, I guess that's a mistake?

Also, you can drop the `binutils` build dep, that's implied with `GCC` (but not with `GCCcore`)